tests/init_test.go

```diff
@@ -65,8 +65,9 @@ var (
 	legacyStateTestDir = filepath.Join(baseDir, "LegacyTests", "Constantinople", "GeneralStateTests")
 	transactionTestDir = filepath.Join(baseDir, "TransactionTests")
 	rlpTestDir         = filepath.Join(baseDir, "RLPTests")
-	difficultyTestDir  = filepath.Join(baseDir, "BasicTests.core-geth")
-	benchmarksDir      = filepath.Join(".", "evm-benchmarks", "benchmarks")
+	// difficultyTestDir  = filepath.Join(baseDir, "BasicTests.core-geth")
+	difficultyTestDir = filepath.Join(baseDir, "BasicTests")
```

Comment:
TODO. 

(Difficulty tests are often core-geth specific because core-geth has lots of logic for difficulties that don't care about the difficulty bomb. A separate dir was used, if I remember right, because they were removed or deprecated upstream... possibly with the "Legacy"/"Constantinople" labels and partitioning that happened upstream a while ago).